sdk/src/client/node_manager/syncing.rs

```diff
@@ -104,7 +104,8 @@ impl ClientInner {
             if let Some((info, _node_url)) = nodes.first() {
                 let mut network_info = self.network_info.write().await;
 
-                network_info.latest_milestone_timestamp = info.status.latest_milestone.timestamp;
+                // TODO change to one of the new timestamps, which ones ?
+                // network_info.latest_milestone_timestamp = info.status.latest_milestone.timestamp;
```

Comment:
just marking this so it doesn't go unnoticed (is this solved with this PR?)

Reply:
I've opened an issue already https://github.com/iotaledger/iota-sdk/issues/647

Reply:
@Alex6323 Please resolve :trollface: